bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -1,7 +1,7 @@
 # Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py
 
 from airflow import DAG
-from airflow.operators.sensors import ExternalTaskSensor
+from airflow.operators.sensors import ExternalTaskSensor, TimeDeltaSensor
```

Comment:
Super nit - can this be `{%+ if task.execution_delay %}, TimeDeltaSensor{% endif %}` to prevent unnecessary changes to all of the dag files?